### PR TITLE
fix: edge case where initializing camera callbacks could not be set

### PR DESCRIPTION
**Pull request by MikeWoots**

### CODE REVIEW

File: android/src/main/java/com/aerobotics/DjiMobile/CameraEventDelegate.java [hunk 1]
```diff
@@ -34,40 +34,49 @@ public void sendEvent(Object payload) {
...
-  };
+  private boolean callbacksSet = false;
 
   private KeyListener CameraConnectedKeyListener = new KeyListener() {
     @Override
     public void onValueChange(@Nullable Object oldValue, @Nullable Object newValue) {
       if (newValue != null && newValue instanceof Boolean) {
-        isCameraConnected = (boolean)newValue;
-        setCameraCallbacks();
+        if ((boolean) newValue && !isCameraConnected) {
```
Comment (nhanders) [inline android/src/main/java/com/aerobotics/DjiMobile/CameraEventDelegate.java:44]: There's no problem with a whitespace between `(boolean) newValue` is there?

File: android/src/main/java/com/aerobotics/DjiMobile/DJIMobile.java [hunk 2]
```diff
@@ -529,7 +545,7 @@ public void onValueChange(@Nullable Object oldValue, @Nullable Object newValue)
           params.putString("fileName", mediaFile.getFileName());
           params.putString("dateCreated", mediaFile.getDateCreated());
           params.putDouble("fileSizeInBytes",mediaFile.getFileSize());
-          sendEvent(SDKEvent.CameraDidGenerateNewMediaFile, params);
+          sendRealTimeEvent(SDKEvent.CameraDidGenerateNewMediaFile, params);
```
Comment (nhanders) [inline android/src/main/java/com/aerobotics/DjiMobile/DJIMobile.java:548]: This is awesome

**Review by nhanders — APPROVED** [review]
Seen most of this in person already, and I'm happy 👍

File: android/src/main/java/com/aerobotics/DjiMobile/CameraEventDelegate.java (hunk 1 above)
Comment (MikeWoots) [reply android/src/main/java/com/aerobotics/DjiMobile/CameraEventDelegate.java:44]: No issue other than inconsistent styling

**Pull request merged**